hugolib/site.go

```diff
@@ -909,10 +909,10 @@ func (s *Site) initializeSiteInfo() {
 		Languages:                      languages,
 		defaultContentLanguageInSubdir: defaultContentInSubDir,
 		GoogleAnalytics:                lang.GetString("GoogleAnalytics"),
-		RSSLink:                        permalinkStr(viper.GetString("RSSUri")),
+		RSSLink:                        permalinkStr(lang.GetString("RSSUri")),
 		BuildDrafts:                    viper.GetBool("BuildDrafts"),
 		canonifyURLs:                   viper.GetBool("CanonifyURLs"),
-		preserveTaxonomyNames:          viper.GetBool("PreserveTaxonomyNames"),
+		preserveTaxonomyNames:          lang.GetBool("PreserveTaxonomyNames"),
```

Comment:
Does this mean that `RSSUri` and `PreserveTaxonomyNames` can modified on a per-language basis?  If so, we should note that in the documentation.